spark-jobs/src/main/scala/za/co/absa/enceladus/standardization/interpreter/stages/TypeParser.scala

```diff
@@ -237,6 +237,12 @@ object TypeParser {
       SchemaUtils.getFieldType(inputFullPathName, origSchema).get
     }
 
+    protected def origTypeButArray: DataType = {
```

Comment:
I got very confused by this name. I thought it might mean "original type... but as an array".
How about `origElemType` for "original element type" or `underlyingType` maybe?
Not an easy thing to name...

Reply:
I was actually hoping for some good name suggestions. Wasn't happy with this one myself (but best I came up with).
It's the `origType` except for array's when it's the `origElementType`.
`underlyingType` is nicely vague - so not misleading. But I am not sure it actually says in any way, what is it. But perhaps better then what I have. 
Let's see if others comment.